tests/system/small/test_remote_function.py

```diff
@@ -484,17 +485,27 @@ def add_one(x):
 
 
 @pytest.mark.flaky(retries=2, delay=120)
-def test_series_map(session_with_bq_connection, scalars_dfs):
```

Comment:
do we still need the old `test_series_map` test?

Reply:
I think `test_series_map_bytes` should be sufficent. We have other tests that check integer input and output with remote_function.